packages/vats/src/types.js

```diff
@@ -1,5 +1,7 @@
 // @ts-check
 
+export {};
```

Comment:
Looks like this is now exported types, so it should be easy to re-export from `index.js`

Reply:
This file is currently still referenced by `vats/exported.js` using `import './src/types.js';`, which is no longer relevant.

Have we also confirmed there were no use of the typedefs in this file that may have silently rolled back to an implicit `any` type?

Reply:
> This file is currently still referenced by vats/exported.js using import './src/types.js';, which is no longer relevant.

Good catch. Removed.

> Have we also confirmed there were no use of the typedefs in this file that may have silently rolled back to an implicit any type?

Good idea. Found some (though I don't know if they were resolving before).